node/README.md

```diff
@@ -18,6 +18,7 @@ PLAID_SECRET='SECRET' \
 PLAID_PUBLIC_KEY='PUBLIC_KEY' \
 PLAID_ENV='sandbox' \
 PLAID_PRODUCTS='transactions' \
+PLAID_COUNTRY_CODES='us,ca,gb,fr,es' \
```

Comment:
This will work since we convert all of these to uppercase in the API, but these are supposed to be ISO 3166-1 alpha-2 country codes which should be uppercase.

Take make this more clear can you change all of the country codes to upper case?